elide-core/src/test/java/com/yahoo/elide/core/lifecycle/LifeCycleTest.java

```diff
@@ -242,6 +242,19 @@ public void testElideGetSparse() throws Exception {
         verify(tx).close();
     }
 
+    @Test
+    public void testElideGetInvalidKey() throws Exception {
+        DataStore store = mock(DataStore.class);
+        Elide elide = getElide(store, dictionary, MOCK_AUDIT_LOGGER);
+
+        MultivaluedMap<String, String> headers = new MultivaluedHashMap<>();
+        headers.putSingle("fields[testModel]", "field");
```

Comment:
We should add multiple invalid to test the output with multiple invalid query params.

Reply:
ok.